### DISCUSSION TITLE
Adding more Audio I/O channels

### DISCUSSION THREAD

**Discussion by Nullifier** [DISTRHO/Cardinal#521] [Ideas]
Hi there. Thank you for all the work you do on this project. 
I've just installed the Cardinal VST to use with Ableton and could not fully utilize my Expert Sleepers ES-9 because there are only 8 I/O channels in Cardinal, yet I'll need to use all 16 I/O of ES-9 for the full potential. Am I missing something or can I solve it with the current version? 
In a perfect world, there would be an interface in Cardinal that will give as much I/O as needed, but I don't know about limitations.

**Comment by skeer562** [comment]
I agree.  Otherwise, I'm just gonna have to bite the bullet and purchase VCV.  I much prefer FOSS.  Nullifier, can you share a picture of your patching as to how you got the 8 channels to work?
